website/api/goldparse.jade

```diff
@@ -151,7 +151,7 @@ p
 
     +row
         +cell #[code entities]
-        +cell iterable
+        +cell list
```

Comment:
This change isn't right. The function accepts any iterable of entities, e.g. a tuple, generator, custom type. It's more correct to say "iterable" here than "list".

Reply:
You're right that "list" is too specific, but "iterable" is incorrect, and I actually ran into a bit of trouble the first time I used this function, prior to reading the source, because I assumed it could take any kind of iterable. It _accepts_ a generator, true, but it reads through the entire collection twice, so by the time it gets to the second loop, the generator will be empty. See https://github.com/explosion/spaCy/blob/386cec1/spacy/gold.pyx